ReleaseNotes.md

```diff
@@ -1,11 +1,42 @@
 # Release Notes
 
-## Introduction
-*Neural Network Compression Framework (NNCF)* is a toolset for Neural Networks model compression.
-The framework organized as a Python module that can be built and used as standalone or within
-samples distributed with the code.  The samples demonstrate the usage of compression methods on
-public models and datasets for three different use cases: Image Classification, Object Detection,
-and Semantic Segmentation.
+## New in Release 2.4.0
+Target version updates:
+- Bump target framework versions to PyTorch 1.13.1, TensorFlow 2.8.x, ONNX opset 13
+- Increased target HuggingFace transformers version for the integration patch to 4.23.1
+
+Features:
+- Official release of the ONNX framework support.
+NNCF may now be used for post-training quantization (PTQ) on ONNX models.
+Added an example script demonstrating the ONNX post-training quantization on MobileNetV2.
+- Common post-training quantization API across the supported frameworks (PyTorch, TensorFlow and ONNX) via the `nncf.quantize(...)` function.
+The parameter set of the function is the same for all frameworks - actual framework-specific implementations are being dispatched based on the type of the model object argument.
+- (PyTorch) Joint Pruning, Quantization and Distillation for Transformers made possible via NNCF - 
+- (PyTorch, TensorFlow) Improved the adaptive compression training functionality to reduce effective training time.
+- (ONNX) Post-processing nodes are now automatically excluded from quantization.
+- (PyTorch - Experimental) Joint Pruning, Quantization and Distillation for Transformers enabled for certain models from HuggingFace `transformers` repo
+
```

Comment:
(ONNX) Added FastBiasCorrection and BiasCorrection methods for Post-Training Quantization

Reply:
(ONNX) Added support of Preset

Reply:
This is all part of PTQ for ONNX.